src/html.rs

```diff
@@ -357,7 +357,7 @@ fn gen_html(
         return;
     }
 
-    let mut f = match File::open(&path) {
+    let mut f = match File::open(path) {
```

Comment:
The changes in this file are unrelated from the PR, could you revert them?
They are good changes, but they should be part of a separate PR.

Reply:
no, cause the CI fails because of this

Reply:
We should land a PR to fix the CI first, and then rebase this on top of master.

Reply:
Do you want to do that or do you prefer I do that?

Reply:
yes would be nice, than i will rebase

Reply:
done, all your conversions are outdated now